### DISCUSSION TITLE
Move printed configuration from pre-execution to pre-prompt

### DISCUSSION THREAD

**Discussion by diegoferigo** [starship/starship#4558] [General]
My prompt inside a docker container looks like the following:

![Screenshot_20221104_093332](https://user-images.githubusercontent.com/469199/199928393-b54335e7-55a7-40ac-833a-9a666126aa5b.png)

I did not find any documentation about how (and, first, if it's possible) to move the output of this module on the first line (pre-prompt), i.e. next to the python info.

**Comment by chipbuster** [comment]
This is controlled by the [prompt format](https://starship.rs/config/#prompt). The whale most likely comes from the `$container` module--you can modify so that this module goes before `$line_break` to get it all on the first line.
